functions/fun10.go

```diff
@@ -0,0 +1,27 @@
+/* using flag package and accessing flag arguments
+using pointers 
+./fun10 a bc def
+./fun10 -s / a bc def ga
+a/bc/def/ga
+./fun10 --help
```

Comment:
--help returns non-zero exit. why?

Reply:
From the documentation , this seems a default behaviour: https://golang.org/pkg/flag/#pkg-variables 
https://golang.org/pkg/flag/#ExitOnError (constant with value as 2 )